docs/general/configuration.md

```diff
@@ -62,4 +62,46 @@ Like using patchwork, strict mode has to be enabled before the WP_Mock framework
 ```php
 WP_Mock::activateStrictMode();
 WP_Mock::bootstrap();
+```
+
+## Extend WP_Mock Test Case
+
+Once you have set up WP_Mock as outlined above, you should use the `WP_Mock\Tools\TestCase` class as your base test case class in your PHPUnit tests. This class extends PHPUnit's own `TestCase` class, with some helper methods but also methods that help WP_Mock to function properly. You should always extend `WP_Mock\Tools\TestCase` instead of `PHPUnit\Framework\TestCase` when using WP_Mock.
```

Comment:
Just one nitpick here ... we should probably soften the "always" language since we mention it's not required a few lines later. Here's a suggested rewording:

```suggestion
Once you have followed the configuration steps as outlined above, you'll want to update your test classes to extend the `WP_Mock\Tools\TestCase` class for the best results when using WP_Mock. This class extends PHPUnit's own `TestCase` class, with some helper and other methods that help WP_Mock to function properly.
```